lib/util.ex

```diff
@@ -17,45 +21,47 @@ defmodule Util do
         # let's just assume there are no more than 10 lol.
         # (At time of writing there are less than 1,000 upcoming tournaments,
         # which can easily fit in only 2 pages.)
-        tourneys = 1..10
-                   |> Enum.map(&get_tourney_page(auth, &1))
-                   |> List.flatten
-        { :ok, %{ tournament_data: tourneys, metadata: %{ updated_at: DateTime.now!("Etc/UTC") |> DateTime.to_unix()}} }
+        tourneys =
+          1..10
+          |> Enum.map(&get_tourney_page(auth, &1))
```

Comment:
In reference to the new comment below, I'm guessing we could maybe do some intermittent checking of the result for each page and halt the looping? Unsure how to do that in Elixir though, or we even want to lol